client/src/document/spec-table.tsx

```diff
@@ -0,0 +1,32 @@
+import { DisplayH2, DisplayH3 } from "./ingredients/utils";
+
+export function SpecificationTable({
+  id,
+  title,
+  isH3,
+  spec_urls,
+  query,
+}: {
+  id: string;
+  title: string;
+  isH3: boolean;
+  spec_urls: [];
```

Comment:
Shouldn't that be:
```suggestion
  spec_urls: string[];
```

Reply:
Also, would you mind looking at renaming the variable to something like `specURLs`?
In `LazyBrowserCompatibilityTable` it's called `dataURL` (not `data_url`).